doc/DeveloperGuide.md

```diff
@@ -87,6 +87,32 @@ Use case ends.
 
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
+  
+#### Use case: Edit person
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to edit a specific person in the list
+4. User input some data that will be updated
```

Comment:
The Use Case seeks to model the interaction between the Actor and the System. In the form of user -> system -> user -> system -> etc.
In this case do you see which line needs to be modified? =)